gapis/server/server.go

```diff
@@ -230,6 +233,14 @@ func (s *server) LoadCapture(ctx context.Context, path string) (*path.Capture, e
 	if _, err = capture.ResolveFromPath(ctx, p); err != nil {
 		return nil, err
 	}
+	// Pre-resolve the dependency graph.
```

Comment:
@AWoloszyn I'm not sure if this is a proper place to insert the code. This makes the function name: 'LoadCapture' not that accurate. If so, we also need to do the same for 'ImportCapture'.

Another place is the capture.go:Import(), just after the `path.Capture` is built. Then we will need to resolve the capture there first, then resolve dependency graph.

Any suggestions?